apps/web-desmos/src/screens/proposal_details/components/overview/index.tsx

```diff
@@ -132,16 +132,20 @@ const Overview: FC<{ className?: string; overview: OverviewType }> = ({ classNam
       />
       <Divider />
       <div className={classes.content}>
-        <Typography variant="body1" className="label">
-          {t('type')}
-        </Typography>
-        <Typography variant="body1">
-          {types.map((type: string) => (
-            <Typography variant="body1" className="value" key={type}>
-              {t(type)}
+        {!!(types.length > 0) && (
```

Comment:
In future don't need to do this `!!(types.length > 0)`, can be directly `types.length > 0`